systemtest/src/test/java/io/strimzi/systemtest/operators/topic/TopicReplicasChangeST.java

```diff
@@ -464,10 +464,6 @@ void setup() {
                                 .addToLimits("memory", new Quantity("1Gi"))
                                 .addToRequests("memory", new Quantity("1Gi"))
                                 .build())
-                            // faster cluster model generation tuning: reduce sampling and metadata refresh intervals
-                            .addToConfig("metric.sampling.interval.ms", 5_000)
-                            .addToConfig("cruise.control.metrics.reporter.metrics.reporting.interval.ms", 5_000)
-                            .addToConfig("metadata.max.age.ms", 4_000)
```

Comment:
Note that these CC turnings were added to make this test faster, to avoid hitting the configured timeout. Before them it was randomly hitting the timeout when running locally.

Reply:
I guess you missed this https://github.com/strimzi/strimzi-kafka-operator/pull/10166/files#diff-7c26cd3eed81421a1e40192f3e1cd533bfd89771e4f2a759eacc1a8a3dd51c3eR254

Reply:
Right, I missed that, thanks. But then you should also get rid of the "editKafka" part (lines 452-454).

Reply:
Don't you have the same config in the link you posted?

Reply:
oh you were talking about code in TopicReplicasChangeST, undrestood now. I will remove it when the pipelines will finish.

Reply:
Yeah, sorry for the confusion.